### PR TITLE
Add recent changes to CHANGELOG.md

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -265,6 +265,7 @@
 * Verify the "first sound" of as an analysis sanity check [#4773](https://github.com/mixxxdj/mixxx/pull/4773)
 * Fix a possible crash when ejecting a track [#11334](https://github.com/mixxxdj/mixxx/pull/11334) [#11257](https://github.com/mixxxdj/mixxx/issues/11257)
 * Add a range limits for beatjump_size of 512 [#11248](https://github.com/mixxxdj/mixxx/pull/11248) [#11203](https://github.com/mixxxdj/mixxx/issues/11203)
+* Slip mode: consider active (regular) loop [#11435](https://github.com/mixxxdj/mixxx/pull/11435) [#6993](https://github.com/mixxxdj/mixxx/issues/6993)
```
Comment (Holzhaus) [inline CHANGELOG.md:268]: Maybe this is better?

```suggestion
* Slip Mode: Preserve active (regular) loop when leaving Slip Mode [#11435](https://github.com/mixxxdj/mixxx/pull/11435) [#6993](https://github.com/mixxxdj/mixxx/issues/6993)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -370,6 +371,7 @@
 * Exclude HID device: ELAN touch screen [#11324](https://github.com/mixxxdj/mixxx/pull/11324) [#11323](https://github.com/mixxxdj/mixxx/issues/11323)
 * Show otherwise hidden HID devices in developer mode  [#11317](https://github.com/mixxxdj/mixxx/pull/11317)
 * Use hidapi's hidraw backend instead of libusb on Linux [#4054](https://github.com/mixxxdj/mixxx/pull/4054)
+* Fix hid controllers: restore legacy passing of HID data in plain array [#11470](https://github.com/mixxxdj/mixxx/pull/11470) [#11461](https://github.com/mixxxdj/mixxx/issues/11461)
```
Comment (Holzhaus) [inline CHANGELOG.md:374]: What does this mean from a user's perspective?

File: CHANGELOG.md [hunk 3]
```diff
@@ -392,6 +394,7 @@
   [#10540](https://github.com/mixxxdj/mixxx/issues/10540)
   [#10539](https://github.com/mixxxdj/mixxx/issues/10539)
 * Controller preferences: add search bars to mapping tables [#11165](https://github.com/mixxxdj/mixxx/pull/11165)
+* Added declarations for engine and controller scripting API [#4759](https://github.com/mixxxdj/mixxx/pull/4759)
```
Comment (Holzhaus) [inline CHANGELOG.md:397]:
```suggestion
* Added TypeScript declarations for engine and controller scripting API to improve IDE code completion during mapping developent [#4759](https://github.com/mixxxdj/mixxx/pull/4759)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -693,6 +702,8 @@
   [#11074](https://github.com/mixxxdj/mixxx/pull/11074)
   [#11238](https://github.com/mixxxdj/mixxx/pull/11238)
   [#11457](https://github.com/mixxxdj/mixxx/pull/11457)
+  [#11562](https://github.com/mixxxdj/mixxx/pull/11562)
+* Windows buildenv: Fix 7-Zip quoting [#11510](https://github.com/mixxxdj/mixxx/pull/11510)
```
Comment (Holzhaus) [inline CHANGELOG.md:706]:
```suggestion
* Windows: Fix extraction of build envornment archive when using 7-Zip [#11510](https://github.com/mixxxdj/mixxx/pull/11510)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -835,20 +846,21 @@
...
-* Switch to C++20 [#4832](https://github.com/mixxxdj/mixxx/pull/4832) [#4803](https://github.com/mixxxdj/mixxx/pull/4803)
 * Windows builds: Sign binary and installer with timestamp and sha256 [#4824](https://github.com/mixxxdj/mixxx/pull/4824) [#4825](https://github.com/mixxxdj/mixxx/pull/4825)
 * Preferences: Added option to keep deck playing on track load [#10944](https://github.com/mixxxdj/mixxx/pull/10944) [#10548](https://github.com/mixxxdj/mixxx/issues/10548)
 * Preferences: always enable Alt shortcut keys [#11145](https://github.com/mixxxdj/mixxx/pull/11145) [#10413](https://github.com/mixxxdj/mixxx/issues/10413)
 * macOS: Add better rasterization of program icon [#10958](https://github.com/mixxxdj/mixxx/pull/10958)
 * Allow building Mixxx on OpenBSD [#11083](https://github.com/mixxxdj/mixxx/pull/11083)
 * Fix autogenerated metainfo [#11383](https://github.com/mixxxdj/mixxx/pull/11383)
+* Fix setting the wrong default cue color. [#11554](https://github.com/mixxxdj/mixxx/pull/11554) [#11260](https://github.com/mixxxdj/mixxx/issues/11260)
```
Comment (Holzhaus) [inline CHANGELOG.md:863]:
```suggestion
* Fix setting the wrong default cue color [#11554](https://github.com/mixxxdj/mixxx/pull/11554) [#11260](https://github.com/mixxxdj/mixxx/issues/11260)
```

File: CHANGELOG.md (hunk 2 above)
Comment (Swiftb0y) [reply CHANGELOG.md]: Mappings using hid-common-packet-parser.js aren't broken in 2.4. this fixes a regression introduced in refactoring that code.

Comment (Holzhaus) [reply CHANGELOG.md]: Ok, the new description is definitely better: https://github.com/mixxxdj/mixxx/pull/11568/commits/ad11a6463f4ccf3a386c54432cde6547507801ee

**Review by Holzhaus — APPROVED** [review]

**Pull request merged**